include/pros/rotation.hpp

```diff
@@ -25,6 +25,7 @@
 #include <iostream>
 
 #include "pros/rotation.h"
+#include "rtos.hpp"
```

Comment:
is this necessary? if not we should prob remove this.

Reply:
Yes, this is used for pros::Mutex

Reply:
Yes, this is used for pros::Mutex.